eth/client.go

```diff
@@ -158,42 +150,20 @@ func NewClient(account accounts.Account, passphrase string, datadir string, back
 
 	transactOpts.GasPrice = gasPrice
 
-	token, err := contracts.NewLivepeerToken(tokenAddr, backend)
+	controller, err := contracts.NewController(controllerAddr, backend)
```

Comment:
Now we can just pass in the controller address and we can get every other contract address via the controller's registry

Reply:
👍